todo_app/templates/index.html

```diff
@@ -7,11 +7,19 @@ <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
 
+  <from action="add Todo" method="post">
+    <label for="name">Enter New Todo: </label>
+    <input type="text" name="title" required />
+    <input type="submit" value="Add New Todo" />
+    </from>
+
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items %}
```

Comment:
I don't see my items appear on screen by default, and I think that's just a mismatch between the expected variable name here (`items`) and what it's passed through as in the route (`app.py` line 14, passes it through as `my_items`):
```suggestion
        {% for item in my_items %}
```